code/_onclick/hud/alert.dm

```diff
@@ -310,7 +310,7 @@ so as to remain in compliance with the most up-to-date laws."
 
 /obj/screen/alert/restrained/buckled
 	name = "Buckled"
-	desc = "You've been buckled to something and can't move. Click the alert to unbuckle unless you're handcuffed."
+	desc = "You've been buckled to something. Click the alert to unbuckle unless you're handcuffed."
```

Comment:
Part of the goal of the alert was to explain why you're unable to move. Removing that seems counterproductive.